src/handlers/deletelink.ts

```diff
@@ -0,0 +1,51 @@
+import {
+	parseDeleteString,
+	constructSlackMessage,
+} from '../utils';
+
+/**
+ * Sends POST to link shortening service
+ *
+ * @param path – parsed path string
+ * @param url - parsed url string
+ */
+const deleteOldUrl = (path: string): Promise<Response> => {
+	const fetchUrl = `https://vhl.ink/${path}`;
+
+	const headers = new Headers({
+		'Content-Type': 'application/x-www-form-urlencoded',
+		'x-preshared-key': SECRET_KEY,
+	});
+
+	return fetch(`${fetchUrl}`, {
+		method: 'DELETE',
+		headers
+	});
+};
+
+/**
+ * Handles shortening links. Expects the text to look like:
```

Comment:
The comment should be adjusted to reflect what the function does, but looks good!